fecgov/openFEC
# Fix COH on election profile pages

## Summary (required)
Cash-on-hand(COH) for 2020 candidates is now displaying the amount on election profile pages. 

- Resolves # https://github.com/fecgov/openFEC/issues/3562

## How to test the changes locally

- checkout the branch `feature/fix-coh-on-election-profile-pages`
- export the `SQLA_CONN` to point to DEV DB
- The `/elections/` endpoint should now display amount against `cash_on_hand_end_period` column.
http://127.0.0.1:5000/v1/elections/?page=1&sort_nulls_last=false&per_page=20&election_full=true&cycle=2020&office=president&sort_null_only=false&sort=-total_receipts&sort_hide_null=false

## Impacted areas of the application
List general components of the application that this PR will affect:

- Website:  https://www.fec.gov/data/elections/president/2020/
- API : https://api.open.fec.gov/v1/elections/?api_key=DEMO_KEY&cycle=2020&election_full=true&office=president&per_page=0&sort_hide_null=true
